Controller/Adminhtml/CrossBorderType/Edit.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * ClassyLlama_AvaTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @copyright  Copyright (c) 2018 Avalara, Inc.
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+namespace ClassyLlama\AvaTax\Controller\Adminhtml\CrossBorderType;
+
+use ClassyLlama\AvaTax\Api\CrossBorderTypeRepositoryInterface;
+use Magento\Framework\Exception\LocalizedException;
+
+class Edit extends \ClassyLlama\AvaTax\Controller\Adminhtml\CrossBorderType
+{
+
+    /**
+     * @var \Magento\Framework\View\Result\PageFactory
+     */
+    protected $resultPageFactory;
+
+    /**
+     * @var CrossBorderTypeRepositoryInterface
+     */
+    protected $crossBorderTypeRepository;
+
+    /**
+     * @param \Magento\Backend\App\Action\Context        $context
+     * @param CrossBorderTypeRepositoryInterface         $crossBorderTypeRepository
+     * @param \Magento\Framework\Registry                $coreRegistry
+     * @param \Magento\Framework\View\Result\PageFactory $resultPageFactory
+     */
+    public function __construct(
+        \Magento\Backend\App\Action\Context $context,
+        CrossBorderTypeRepositoryInterface $crossBorderTypeRepository,
+        \Magento\Framework\Registry $coreRegistry,
+        \Magento\Framework\View\Result\PageFactory $resultPageFactory
+    )
+    {
+        parent::__construct($context, $coreRegistry);
+
+        $this->resultPageFactory = $resultPageFactory;
+        $this->crossBorderTypeRepository = $crossBorderTypeRepository;
+    }
+
+    /**
+     * Edit action
+     *
+     * @return \Magento\Framework\Controller\ResultInterface
+     */
+    public function execute()
+    {
+        // 1. Get ID and create model
```

Comment:
There is no `// 2. ****` comment, so just remove the `1.` and `3.` from the comment below